src/layer/form-widgets/index.md

```diff
@@ -0,0 +1,298 @@
+# Setting Up Widgets in Attributes Form
+
+[[toc]]
+
+Capturing field data often requires filling some attributes in the form to record the properties of surveyed points, lines or polygons. Attribute forms can simplify the data entry and even ensure to some extent that the correct information is filled in. Here, we will explore the possibilities of various QGIS widgets that can be used in <MobileAppName />. 
+
+Extra configuration of attribute forms can be done to make data collection easier and more consistent, such as using default values, conditional visibility or constraint enforcement. These topics are covered in [Attributes Form Configuration](../form-configuration/) and [Attributes Form Layout](../form-layout/).
+
+<YouTube id="jc4p1PpXj3k" />
+
+## Widget gallery
+Attributes forms can be configured using <QGISHelp ver="latest" link="user_manual/working_with_vector/vector_properties.html#edit-widgets" text="QGIS widget types" /> in the **Attributes Form** tab in the **Layer Properties**.
+
+A number of QGIS widget types can be used in <MobileAppName />, such as *Text Edit*, *Range* widget for numbers, *Date/Time* widget with calendar popup, *Value Map* and *Value Relation* to select options from a drop-down menu, or the *Attachment* widget that is used to save photos. The widget has to be compatible with the data type of a field. 
```

Comment:
I find it unnecessary to list a subset of widgets here.
```suggestion
The widget has to be compatible with the data type of a field. 
```

However, the last sentence about compatibility with the data type of a field is quite important. Let's file a standalone ticket to explain this better - which data types are compatible with which widgets

Reply:
I created a new ticket: https://github.com/MerginMaps/docs/issues/501 , we can discuss how/where to include this information.